cruddy/__init__.py

```diff
@@ -185,6 +215,29 @@ def _new_response(self):
     def _get_ts(self):
         return int(time.time() * 1000)
 
+    def query(self, query_string):
+        response = self._new_response()
+        if '=' not in query_string:
+            response.status = 'error'
+            response.error_type = 'InvalidQuery'
+            response.error_message = 'Only the = operation is supported'
+        else:
```

Comment:
It may be worth checking for `==` as I'm sure someone will make the typo at some point. Maybe even silently convert `==` to `=` and proceed as normal?